src/marketing/price-rule-catalog-scheduled-changes.md

```diff
@@ -3,12 +3,87 @@ conditions: Default.EE-B2B
+The Scheduled Changes box appears at the top of the page when a new price rule is saved or updated. Catalog price rules can be applied on schedule as part of a campaign, and grouped with other content changes. You can create a new campaign based on scheduled changes to a price rule, or apply the changes to an existing campaign.
 
 If there are multiple price rules running in the same campaign, the Priority setting of the price rule determines which rule takes precedence. To learn more, see [Content Staging]({% link cms/content-staging.md %}).
 
 {: .bs-callout .bs-callout-info}
 If a campaign that includes a price rule is initially created without an end date, the campaign cannot be edited later to include an end date. It is recommended that you either add an end date when you create the campaign, or create a duplicate version of the existing campaign and add the end date to the duplicate as needed.
 
-![Catalog price rule - scheduled changes]({% link images/images-ee/content-staging-price-rule-catalog-scheduled-change.png %}){: .zoom}
+![Catalog price rule - scheduled changes]({% link images/images-ee/price-rule-catalog-scheduled.png %}){: .zoom}
 _Scheduled Changes_
+
+## Schedule an update to a catalog price rule
+
+1. On the _Admin_ sidebar, go to **Marketing** > _Promotions_ > **Catalog Price Rule**.
+
+1. Open the rule in edit mode.
+
+1. In the **Scheduled Changes** box at the top of the page, click <span class="btn">Schedule New Update</span>.
+
+1. With the **Save as a New Update** option selected, do the following:
+
+    - In the **Update Name** field, enter a name for the update to the rule.
+
+    - Enter a brief **Description** of the update, including how or why it is applied.
+
+    - Use the Calendar (![]({% link images/images/btn-calendar.png %}){: .Inline}) to choose the **Start Date** and **End Date** for the scheduled change to be in effect. To create an open-ended change, leave the End Date blank.
+
+    {:.bs-callout .bs-callout-info}
+    You cannot add an end date cannot later to a price rule that was originally created without one. In such a case, it is necessary to create a duplicate rule that includes the end date that is needed.
+
+    ![Catalog price rules - new scheduled changes]({% link images/images-ee/price-rule-catalog-schedule-update.png %}){: .zoom}
+    _Add Scheduled Changes_
+
+1. Scroll down to the **Rule Information** section and make the changes for the rule.
+
+   You can schedule a change for any of the rule parameters, including the websites (scope)/customer groups for the rule, conditions of the rule, and actions applied by the rule. Fore more information, see [Creating a Catalog Price Rule]({% link marketing/price-rules-catalog-create.md %}).
+
+    {:.bs-callout .bs-callout-info}
+    If you make a change to any of the rule information parameters, make sure that the **Status** is set as `Active` if you want the change to result in an actively applied rule.
```

Comment:
It's possible that you want to disable an existing rule during a campaign. So I would  rewrite the last clause to say make sure the status is correct.